operator/webserver.py

```diff
@@ -41,12 +67,50 @@ def mission_stop():
             # wait for it to finish
             interfaceprocess.wait()
 
-    return {'message': 'Mission stopped'}
+    car_controls.brake = 1
+    client.setCarControls(car_controls)
+    car_controls.brake = 0 #remove brake
+
+    log = '{}: {}'.format(str(datetime.now()), 'Mission stopped.')
+    logs.append(log)
+    del logs[:]
+
+    log_file.write(log + '\n')
+    log_file.close()
+
+    return {'response': log}
 
 @app.route('/mission/reset', methods=['POST'])
 def mission_reset():
     client.reset()
```

Comment:
Can we add a check for running client? Like, if no AS is connected please don't add the reset message because it's weird.

Reply:
The stop message you mean?

Reply:
If that is what you mean, this has been fixed. You can still reset the car if there is no process running.